bmfuncts/coupling_analysis.py

```diff
@@ -276,35 +326,22 @@ def coupling_analysis(institute, org_tup, wf_path,
         The dict keyed by countries and valued by the list of normalyzed affiliation types \
         to be corrected).
     """
-    # Setting input-data path
-    saved_results_path = set_saved_results_path(wf_path, datatype)
-
-    # Setting aliases from globals
-    analysis_folder_alias = bm_pg.ARCHI_YEAR["analyses"]
-    inst_analysis_folder_alias = bm_pg.ARCHI_YEAR["institutions analysis"]
-    institutions_folder_alias = bm_pg.ARCHI_INSTITUTIONS["root"]
-    inst_types_file_base_alias = bm_pg.ARCHI_INSTITUTIONS["inst_types_base"]
+    # Setting parameters values from params_list
+    institute, org_tup, wf_path, datatype, corpus_year = params_list
 
-    # Setting useful file names
-    inst_types_file_alias = institute + "_" + inst_types_file_base_alias
+    # Setting input-data path
+    final_results_path = set_results_folder_path(wf_path, datatype)
 
     # Setting useful paths
-    year_folder_path = wf_path / Path(str(year))
-    analysis_folder_path = year_folder_path / Path(analysis_folder_alias)
-    inst_analysis_folder_path = analysis_folder_path / Path(inst_analysis_folder_alias)
-    institutions_folder_path = wf_path / Path(institutions_folder_alias)
-    inst_types_file_path = institutions_folder_path / Path(inst_types_file_alias)
+    files_list, folders_list, paths_list = _set_co_files_params(institute, wf_path, corpus_year)
+    analysis_folder_name, inst_analysis_folder_name = folders_list
+    (analysis_folder_path, inst_analysis_folder_path, institutions_folder_path,
```

Comment:
```suggestion
    (analysis_folder_path, inst_analysis_folder_path, _,
```
-> the variable "institutions_folder_path" is not used afterwards

Reply:
Corrected in next commit. Thanks